### ISSUE TITLE
Cannot show LAMMPS commands properly

### ISSUE THREAD

**Issue by CDmaterial** [brunoduran/language-lammps#2]
For example, I tried the following text:
```
units 		metal
boundary	p	p	p
atom_style	atomic
```
They are typical LAMMPS words, but they did not highlighted in my case after installing this package. Is there anything that I need to add at the very beginning?

**Comment by duran** [comment]
Hi @CDmaterial , 

Sorry about getting to you so late.

I have download Atom and installed it, for me it works perfectly fine. Have  you checked that Atom is recognizing the file as a LAMMPS file? It needs to have the `.lmp` extension, otherwise you have to choose the format manually in the bottom right

**Comment by CDmaterial** [comment]
Hi,

Thank you for the reply. I am wondering why it need a .lmp extension? I thought this code is to show the lammps commands for input files.

Thanks,
Cong

**Comment by duran** [comment]
Hi Cong,

> I am wondering why it need a .lmp extension?

Atom uses the extension of the file to decide which kind of file is. So files ending in `.py` are Python, `.js` JavaScript etc.

LAMMPS scripts does not have an official extension, but some users ([like this one](https://github.com/lammps/lammps/blob/master/examples/USER/fep/CC-CO/bar01/in.bar01.lmp)) adopted `.lmp`, `.lmps` or starts the filename with `in.`.

When I made the plugin I chose to highlight automatically `.lmp`, `.lmps` and `.lammps`.
